cob_mimic/src/mimic_node.cpp

```diff
@@ -174,18 +185,21 @@ class Mimic
     bool service_cb_mimic(cob_mimic::SetMimic::Request &req,
                           cob_mimic::SetMimic::Response &res )
     {
+        service_active_ = true;
         blinking_timer_.stop();
 
         res.success = set_mimic(req.mimic, req.repeat, req.speed);
         res.message = "";
 
         if(req.mimic != "falling_asleep" && req.mimic != "sleeping")
             blinking_timer_ = nh_.createTimer(ros::Duration(real_dist_(gen_)), &Mimic::blinking_cb, this, true);
+        service_active_ = false;
         return true;
     }
 
     bool set_mimic(std::string mimic, int repeat, float speed, bool blocking=true)
     {
+        active_mimic_= "Mimic: "+ mimic +", repeat: "+ std::to_string(repeat) +", speed: "+ std::to_string(speed) +", blocking: "+ std::to_string(blocking);
```

Comment:
solve it as in https://github.com/ipa320/cob_driver/pull/393/files#diff-942a6ce5dfeab4525f7e1b076370b5ecR99 (without needing `add_compile_options(-std=c++11)`)